networksurvey/src/androidTest/java/com/craxiom/networksurvey/screens/BottomMenuBar.java

```diff
@@ -8,21 +8,21 @@
 {
     public static void clickWifiMenuOption()
     {
-        clickOn(R.id.main_wifi_fragment);
+        clickOn(R.id.wifi);
```

Comment:
The other android tests failed for me without these changes. They were failing to find the `main_*_fragment`s.

Reply:
Yeah, that is my fault. I broke those tests and only ran unit tests last time, I forgot to run the connected device tests. I can work on fixing them.